actions/generator/generic/create-base64-subjects-from-file/action.yml

```diff
@@ -65,7 +65,12 @@ runs:
         set -euo pipefail
 
         object="{\"artifact_name\": \"${UNTRUSTED_ARTIFACT_NAME}\", \"sha256\": \"${SHA256}\", \"filename\": \"${UNTRUSTED_FILENAME}\"}"
-        echo "$object" | jq
-        base64_object=$(echo "$object" | base64 -w0)
 
+        if test "$(uname)" = "Darwin"
+        then
+            base64_object=$(echo "$object" | base64)
```

Comment:
I think `base64` on MacOS just doesn't have the `-w` option so we'll still need to remove the newlines as GitHub Actions doesn't support newlines in inputs.

We might be able to use `tr` instead regardless of the `RUNNER_OS`?

```shell
base64_object=$(echo "$object" | base64 | tr -d \\n)
```

Reply:
@ianlewis From the linked issue:

> The -w0 flag is used to avoid line wraps. The equivalent flag on macOS would be -b0, but that is already the default

Is that sufficient, in your view, or should we add a second check?